src/_positioning.scss

```diff
@@ -0,0 +1,54 @@
+@import '~bootstrap/scss/mixins/clearfix';
+@import '~bootstrap/scss/mixins/size';
+
+$z-indexes: (
+  'modal',
+  'header',
+  'body',
+  'footer'
+) !default;
+
+@function z($name) {
+  @if index($z-indexes, $name) {
+    @return (length($z-indexes) - index($z-indexes, $name)) + 1;
+  } @else {
+    @warn 'There is no item "#{$name}" in this list; choose one of: #{$z-indexes}';
+    @return null;
+  }
+}
+
+@mixin pseudo($display: block, $pos: absolute, $content: '') {
```

Comment:
I think we can remove prop `position` from this mixin and use mixins `absolute` or `relative` for pseudo elements